specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-09-02-preview/managedClusters.json

```diff
@@ -3562,6 +3585,48 @@
       },
       "description": "Contains the IPTag associated with the object."
     },
+    "PortRange": {
+      "type": "object",
+      "description": "The port range.",
+      "properties": {
+        "portStart": {
+          "type": "integer",
+          "format": "int32",
+          "minimum": 1,
+          "maximum": 65535,
+          "description": "The minimum port that is included in the range. It should be ranged from 1 to 65535, and be less than or equal to `portEnd`."
+        },
+        "portEnd": {
+          "type": "integer",
+          "format": "int32",
+          "minimum": 1,
+          "maximum": 65535,
+          "description": "The maximum port that is included in the range. It should be ranged from 1 to 65535, and be greater than or equal to `portStart`."
+        },
+        "protocol": {
+          "type": "string",
+          "description": "The network protocol of the port.",
+          "enum": [
+            "tcp",
+            "udp"
+          ],
+          "x-ms-enum": {
+            "name": "Protocol",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "tcp",
```

Comment:
I missed that ideally these would be Tcp and Udp (or TCP and UDP), rather than the lowercase forms. _Probably_ too late to change but if it's not, feel free to change it.

Sorry for the miss, that's on me!

Reply:
fixed. change to `TCP` / `UDP`